core/scale/scale_encoder_stream.hpp

```diff
@@ -120,12 +120,12 @@ namespace kagome::scale {
               typename = std::enable_if_t<std::is_integral<I>::value>>
     ScaleEncoderStream &operator<<(T &&v) {
       // encode bool
-      if constexpr (std::is_same<I, bool>::value) {
+      if constexpr (std::is_same<I, bool>::value) {  // NOLINT
         uint8_t byte = (v ? 1u : 0u);
         return putByte(byte);
       }
       // put byte
-      if constexpr (sizeof(T) == 1u) {
+      if constexpr (sizeof(T) == 1u) {  // NOLINT
```

Comment:
for every NOLINT you added please add warning type, e.g. 
`// NOLINT(some-shitty-warning)` so readers in future can understand what kind of warning was suppressed